OpenRA.Mods.Common/Traits/Armament.cs

```diff
@@ -46,10 +49,8 @@ public class ArmamentInfo : UpgradableTraitInfo, ITraitInfo, Requires<AttackBase
 
 		[Desc("Muzzle flash sequence to render")]
 		public readonly string MuzzleSequence = null;
-
```

Comment:
New lines are nice. Please separate all these fields with new lines while you are at it.